src/server/db/migration/migrations/sqls/20221007133533-user-role-on-conflict-up.sql

```diff
@@ -0,0 +1 @@
+alter table public.users_role add constraint user_country_role unique (user_id, country_iso, role);
```

Comment:
isn't user_id, country_iso unique constraint enough ?

Reply:
actually the unique constraint should be: user_id, assessment_id, cycle_uuid, country_iso .
can u verify that?

Reply:
you're right